src/subscription_manager/migrate/migrate.py

```diff
@@ -945,13 +954,15 @@ def validate_options(options):
         if options.environment:
             system_exit(os.EX_USAGE, _("The --activation-key and --environment options cannot be used together."))
         if options.destination_user or options.destination_password:
-            system_exit(os.EX_USAGE, _("The --activation-key option precludes the use of --destination-user and --destination-password"))
+            system_exit(os.EX_USAGE,
+                        _("The --activation-key option precludes the use of --destination-user and "
+                          " --destination-password"))
         if not options.org:
             system_exit(os.EX_USAGE, _("The --activation-key option requires that a --org be given."))
 
     if options.service_level and not options.auto:
         # TODO Need to explain why this restriction exists.
-        system_exit(os.EX_USAGE, _("The --servicelevel and --no-auto options cannot be used together."))
+        system_exit(os.EX_USAGE, _("The --service-level and --no-auto options cannot be used together."))
```

Comment:
good change, however unrelated; better do it in an own PR

Reply:
Changed back and made a note so I can open a PR